milvus/metadata.csv

```diff
@@ -0,0 +1,387 @@
+milvus.cgo.active_future_total,gauge,,,,Total number of active futures.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.bucket,count,,,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.count,count,,,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_duration_seconds.sum,count,,second,,Histogram of cgo call duration in seconds.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.bucket,count,,,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.count,count,,,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.cgo_queue_duration_seconds.sum,count,,second,,Duration of cgo call in queue.,0,milvus,,,
+milvus.cgo.running_cgo_call_total,gauge,,,,Total number of running cgo calls.,0,milvus,,,
+milvus.datacoord.channel_checkpoint_unix_seconds,gauge,,,,Channel checkpoint timestamp in unix seconds,0,milvus,,,
+milvus.datacoord.collection_num,gauge,,,,Number of collections,0,milvus,,,
+milvus.datacoord.consume_datanode_tt_lag_ms,gauge,,millisecond,,Now time minus tt per physical channel,0,milvus,,,
+milvus.datacoord.datanode_num,gauge,,,,Number of data nodes,0,milvus,,,
+milvus.datacoord.import_tasks,gauge,,,,The import tasks grouping by type and state,0,milvus,,,
+milvus.datacoord.index.node_num,gauge,,,,Number of IndexNodes managed by IndexCoord,0,milvus,,,
+milvus.datacoord.index.req.count,count,,,,Number of building index requests,0,milvus,,,
+milvus.datacoord.index.task_count,gauge,,,,Number of index tasks of each type,0,milvus,,,
+milvus.datacoord.segment_num,gauge,,,,Number of segments,0,milvus,,,
+milvus.datacoord.stored.binlog_size,gauge,,byte,,Binlog size of healthy segments,0,milvus,,,
+milvus.datacoord.stored.index_files_size,gauge,,byte,,Index files size of the segments,0,milvus,,,
+milvus.datacoord.stored.rows_num,gauge,,,,Number of stored rows of healthy segment,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.bucket,count,,,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.count,count,,,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.task_execute_max_latency.sum,count,,millisecond,,Latency of task execute operation,0,milvus,,,
+milvus.datacoord.watched_dml_chanel_num,gauge,,,,The num of dml channel watched by datanode,0,milvus,,,
+milvus.datanode.autoflush_buffer_op.count,count,,,,Count of auto flush buffer operations,0,milvus,,,
+milvus.datanode.consume.bytes.count,count,,,,,0,milvus,,,
+milvus.datanode.consume.msg.count,count,,,,Count of consumed msg,0,milvus,,,
+milvus.datanode.consume.tt_lag_ms,gauge,,millisecond,,Now time minus tt per physical channel,0,milvus,,,
+milvus.datanode.encode_buffer_latency.bucket,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.count,count,,,,Latency of encode buffer data,0,milvus,,,
+milvus.datanode.encode_buffer_latency.sum,count,,millisecond,,Latency of encode buffer data,0,milvus,,,
```

Comment:
The description field should explain what the difference is between these three metrics.